align-bwamem.wdl

```diff
@@ -2,7 +2,7 @@ version 1.0
 
 import "tasks/bwa.wdl" as bwa
 import "tasks/samtools.wdl" as samtools
-import "tasks/bwa.wdl" as bwa
+import "tasks/common.wdl" as common
```

Comment:
Maybe better to move the BWA index struct to BWA because that makes most sense.

Reply:
BwaIndex is already inside bwa but IndexedBam is in common